destinations/zendesk/index.js

```diff
@@ -0,0 +1,59 @@
+async function track(event, settings) {
+  const endpoint = `https://${settings.subdomain}.zendesk.com/api/v2/tickets.json`
+  
+  if (event.event = "Application Submitted") {
+    
+    const newTicket = {
+      "ticket": {
+        "subject":  `Application Review: ${event.properties.display_name}`,
+        "comment":  { 
+          "body": `Type: ${event.properties.component_type}
```

Comment:
What's the reason to include these all in a string inside of the body? Any way we could add some details about how this shows up downstream?

Reply:
That's the only area you can include custom information - as an internal note in Zendesk.